pom.xml

```diff
@@ -260,6 +261,24 @@ THE SOFTWARE.
             <version>2.2</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.modules</groupId>
+            <artifactId>sshd</artifactId>
+            <version>2.3</version>
+            <scope>test</scope><!-- from git-server via workflow-cps-global-lib and otherwise unavailable during tests -->
```

Comment:
Sigh.

Maybe it would be best to make `workflow-cps-global-lib` be just its original function—global libraries using `git-server`—and introduce a new plugin `pipeline-libs` with the newer external libraries stuff. `workflow-cps-global-lib` would then depend on `pipeline-libs` to pick up some common infrastructure, and could be deprecated (dropped from `workflow-aggregator`, etc.) and not used in tests of other plugins. People with `workflow-cps-global-lib` already installed would see no change in behavior, but new installations would not normally get the Git service.